src/main/java/org/primefaces/component/timeline/TimelineRenderer.java

```diff
@@ -207,12 +207,8 @@ protected void encodeScript(FacesContext context, Timeline timeline) throws IOEx
         if (timeline.getMaxHeight() != null) {
             wb.attr("maxHeight", timeline.getMaxHeight());
         }
-        if (timeline.getHorizontalScroll() != null) {
-            wb.attr("horizontalScroll", timeline.getHorizontalScroll());
-        }
-        if (timeline.getVerticalScroll() != null) {
-            wb.attr("verticalScroll", timeline.getVerticalScroll());
-        }
+        wb.attr("horizontalScroll", timeline.isHorizontalScroll());
+        wb.attr("verticalScroll", timeline.isVerticalScroll());
```

Comment:
```java
 wb.attr("verticalScroll", timeline.isHorizontalScroll(), false);
 wb.attr("verticalScroll", timeline.isVerticalScroll(), false);
```
This won't send the JSON through network traffic if the value is false which is the default.

Reply:
Thanks for your advice. I changed.